build/Uno.WinUI.MediaPlayer.Skia.Gtk.nuspec

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+	<metadata minClientVersion="5.0">
+		<id>Uno.UI.MediaPlayer.Skia.Gtk</id>
+		<version>0.0.1</version>
+		<authors>nventive</authors>
+		<icon>uno.png</icon>
+		<projectUrl>https://github.com/unoplatform/uno</projectUrl>
+		<license type="expression">Apache-2.0</license>
+		<description>This package provides support for MediaPlayerElement for Uno Platform Skia+Gtk apps</description>
+		<copyright>nventive</copyright>
+		<repository type="git" url="https://github.com/unoplatform/uno.git" branch="$branch$" commit="$commitid$" />
+		<dependencies>
+			<dependency id="Uno.WinUI" version="to-be-set-by-ci" />
```

Comment:
Should we include the other dependencies?

https://github.com/unoplatform/uno/blob/5728bc3bc4d5e1af49bc53ecae978ccdd1c71d50/src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/Uno.UI.MediaPlayer.Skia.Gtk.csproj#L33-L36